src/otx/algo/classification/multiclass_models/mobilenet_v3.py

```diff
@@ -45,6 +45,7 @@ def __init__(
         label_info: LabelInfoTypes,
         data_input_params: DataInputParams,
         model_name: str = "mobilenetv3_large",
+        freeze_backbone: bool = False,
```

Comment:
Constructors of all the OTX public model in this folder should be updated, like it's done in this file